pkg/api/heartbeat.go

```diff
@@ -65,8 +65,6 @@ func (c *Client) sendHeartbeats(url string, heartbeats []heartbeat.Heartbeat) ([
 	}
 	defer resp.Body.Close()
 
-	log.Debugf("request headers: %+v", req.Header)
```

Comment:
oops! 👍

Reply:
We can add logging for headers back if we loop over the headers and hide the `Authorization` header's value.